configure.ac

```diff
@@ -6,7 +6,7 @@
 
 #	Process this file with autoconf to produce a configure script.
 
-AC_PREREQ([2.59])
+AC_PREREQ([2.64])
```

Comment:
NOTE for myself: the version of autoconf shipped as part of RHEL6 is 2.63.
The time for thinking about preparing weekly tarball with generated configure comes.